int32 x_movement
int32 z_movement
int32 grasp_angle
int32 grabe_wide
